python/ray/includes/metric.pxi

```diff
@@ -12,6 +12,7 @@ from libcpp.string cimport string as c_string
 from libcpp.unordered_map cimport unordered_map
 from libcpp.vector cimport vector as c_vector
 
+# TODO: is this actually used anywhere?? all the TagKey references I see are to the protobuf. You can't even instantiate it from purepython!
```

Comment:
That seems right, I created a PR here to remove it https://github.com/ray-project/ray/pull/56939, let's see if it passes the CI :)